lib/json_schema_form/umm_json_form.rb

```diff
@@ -152,7 +152,10 @@ def convert_key_to_type(value, type)
     return (value.casecmp('true') >= 0 ? true : false) if type == 'boolean'
 
     # Numbers
-    return UmmUtilities.convert_to_integer(value) if type == 'number'
+    return UmmUtilities.convert_to_number(value) if type == 'number'
+
+    # Integers
+    return UmmUtilities.convert_to_integer(value) if type == 'integer'
```

Comment:
I don't think there are any integer fields right now, but previously this was converting to an integer if the type was number, which was causing problems. Seemed like a good idea to cover all the bases